v8/plantuml/README.md

```diff
@@ -0,0 +1,30 @@
+This plugin converts [PlantUML](https://plantuml.com/) files.
+
+The default configuration will output all `*.puml` files found under the `pages` dir as SVG files.
+
+Developed against PlantUML version 1.2020.24.  Probably works with some earlier versions.
+
+# Unicode
+
+The plugin expects PlantUML files to be encoded with UTF-8.
+
+# Known Issues
+
+- It's slow!  Every PlantUML rendering launches a new Java process, on my laptop it takes 4-8 seconds per file.
+  I have some ideas to speed this up, and they may be available in future plugin versions.
+
+- Changes to files included via `!include ...` or via a pattern (e.g. `-Ipath/to/*.iuml`) will NOT trigger a rebuild.
+  Instead, if you include them explicitly in `PLANTUML_ARGS` (e.g. `-Ipath/to/foo.iuml`) then they will trigger a
+  rebuild.
+
+- `nikola auto` does not watch dirs in `PLANTUML_FILES` or files included via `PLANTUML_ARGS` / `!include`.
```

Comment:
Perhaps in a separate PR we could add an`EXTRA_WATCHED_FOLDERS` config option or `BasePlugin.register_watched_folder()` function to Nikola?

Reply:
:+1: from me for `register_auto_watched_folder`, the config option is not necessary.